src/molecules/Toggle/Toggle.css

```diff
@@ -0,0 +1,95 @@
+.Toggle {
+  margin: 1rem;
+  cursor: pointer;
```

Comment:
Can you check on how this style is being applied? I get a `pointer` cursor when hovering to the side of your Toggle, but not when hovering the UI.

Also - could you please do a quick pass through your CSS again to remove comments and alphabetize?